### PR TITLE
Feat/quoteimage crosslinks

### LINKED ISSUE #231: Module_19_QuoteImage — crosslinks
We need crosslinks 

## Actual behaviour!

[bildschirmfoto 2017-04-03 um 10 16 56](https://cloud.githubusercontent.com/assets/23631312/24600972/64f7caa8-1857-11e7-9811-f3c34bc035aa.png)

## Expected behaviour

**Name** links to Ina Feistritzers Product People Page
**More about our Team** links to About-Who Page

module_quoteimage

Arrow 24 px 
Text Style: Credit-Maison / All Caps 

bildschirmfoto 2017-04-03 um 10 30 23

bildschirmfoto 2017-04-03 um 10 35 17

### PR DESCRIPTION
**Pull request by Alexpeschel**
fix #231

### CODE REVIEW

File: src/css/basic.less [hunk 1]
```diff
@@ -57,6 +57,25 @@ a {
...
+	&__name,
+	&__title {
+		display: block;
+	}
+
+	&__link {
+		position: relative;
+		
+		.icon {
+			position: absolute;
```
Comment (marionebl) [inline src/css/basic.less]: Are you absolutely sure we need to resort to absolute positioning here? I'd imagine `display: flex` on `cite__link` should do the trick, too.

File: src/templates/module_19_quoteimage/template.html [hunk 2]
```diff
@@ -8,7 +8,50 @@
 			className="h3"
 			text={props.copy}
 			/>
-		<cite className="cite xscopy">{props.credits}</cite>
+
+		<cite className="cite xscopy">
+				{
+					props.creditsLink &&
+						<a className="link" href={props.creditsLink}>
+							<svg className="icon" viewBox="0 0 20 20">
```
Comment (marionebl) [inline src/templates/module_19_quoteimage/template.html]: Please factor out `<Icon/>` to its own element. Consider those other callsites for `<Icon />`:
* [module_08_links](https://github.com/sinnerschrader/website/blob/53920222d796a6a0acdcf0b8fb96b9349242f324/src/templates/module_08_links/template.html)
* [element_smn](https://github.com/sinnerschrader/website/blob/53920222d796a6a0acdcf0b8fb96b9349242f324/src/templates/element_smn/template.html)

Also use `props.base` analogous to [element_smn](https://github.com/sinnerschrader/website/blob/ea0d6db919bf6e62d56f1395ed00021d7915009f/src/templates/element_smn/template.html#L31)

File: src/css/_helper.less [hunk 3]
```diff
@@ -397,3 +397,7 @@
 	}
 
 }
+
+.linebreak {
+	display: block;
+}
```
Comment (marionebl) [inline src/css/_helper.less]: Missing finishing newline

File: src/css/basic.less (hunk 1 above)
Comment (Alexpeschel) [inline src/css/basic.less]: It has to be position absolute so the icon is not affecting the cite text.

**Pull request merged**